README.md

```diff
@@ -80,32 +80,39 @@ pip uninstall jupyterlab_accessible_themes
    git clone https://github.com/Quansight-Labs/jupyterlab-accessible-themes.git
    ```
 
-2. Change to the `jupyterlab_accessible_themes` directory
+2. Change to the `jupyterlab-accessible-themes` directory
 
    ```bash
-    cd jupyterlab_accessible_themes
+    cd jupyterlab-accessible-themes
    ```
 
-3. Install the package in development mode
-
-   ```bash
-   pip install -e .
-   ```
-
-4. Now you'll need to link the development version of the extension to JupyterLab and rebuild the Typescript source:
+3. Install the node dependencies and build the extension
 
    ```bash
+   # Install jupyterlab 3.x
+   pip install jupyterlab < 4.0
```

Comment:
One dependency here is not like the others - i.e. jupyterlab is not a node dependency per se

Reply:
I have one idea on how to improve the install instructions, let me know if you have more ideas we can include 

Basically in this extension we have a chicken and egg problem with the installation. Ideally, the `pip install` command should do everything, **but** we also need to build all the different sub-packages (one per theme) for it to work. At the end, this means that the developer needs to at least have `jlpm` in their system, which basically comes with jupyterlab. This step needs to be done only once in the environment, and it may be optional if the developer is already using an environment that has jupyterlab installed.

I will move this instruction as an optional item, let me know if it works better.